services/api-server/src/simcore_service_api_server/core/health_checker.py

```diff
@@ -69,7 +69,9 @@ async def teardown(self):
 
     @property
     def healthy(self) -> bool:
-        return self._health_check_failure_count <= self._allowed_health_check_failures
+        return self._rabbit_client.healthy and (
```

Comment:
Make sure to add a link to this PR in this condition OR (even better) to add a test. Otherwise in the new future someone could change this

Reply:
Done, thanks